### DISCUSSION TITLE
Getting error : microsoft.ul.xaml nuget requires target platform xxx

### DISCUSSION THREAD

**Discussion by robsadams** [microsoft/react-native-windows#12604] [General]
I am getting the error message: Microsoft.UI.Xaml nuget requires targetplatformminversion >= 10.0.17763.0 (current project is 17134).... I can't for the life of me figure out how to change the build OS version.... I have searched all over the web and most say to change the build configuration.  But the build menu in visual studio does not appear when doin react-native-windows projects from what I can tell.  I have tried manually updating the myproject.sln file and the Microsoft.ReactNative.Mnanged.csproj file... this did not seem to work... any ideas?

**Comment by ghost** [comment]
I assume that from your question, you might not have experience with Visual Studio. Therefore, I've recorded a short tutorial outlining the steps you should follow.

Navigate to the ```windows``` folder within your project and look for a file named ```YourProjectName.sln```, in my case, it's ```lottieTest.sln```.

Note: My Visual Studio is in pt-BR, so you may need to use a translation tool.

Let me know if this works for you.🤔

https://github.com/microsoft/react-native-windows/assets/92493696/1368ab9a-b4d8-4147-a816-7ec34e8c7b3f
